src/sentry/runner/commands/workstations.py

```diff
@@ -0,0 +1,576 @@
+    elif info.state == WorkstationState.STOPPED or info.state == WorkstationState.STOPPING:
+        _sync_gcloud_workstation_cmd(["workstations", "start", name, "--async"], scope=conf)
+        _notify(f"Starting up your workstation {name}, this could take a few minutes...")
+        # A bit hacky, since we can stack overflow if `TIMEOUT/POLLING_INTERVAL` is large enough.
+        # Solution: just don't make it too large ;)
+        time.sleep(POLLING_INTERVAL)
+        return _connect(name, conf, started_at)
+    elif info.state == WorkstationState.STARTING:
+        # Wait for an already requested startup to complete.
+        _notify(f"Waiting for workstation {name} to start up, this could take a few minutes...")
+        # Ditto re: hackiness (see case above).
+        time.sleep(POLLING_INTERVAL)
+        return _connect(name, conf, started_at)
+    elif info.state == WorkstationState.RUNNING:
+        pass
+
+    # The workstation has started, but that is not sufficient to ensure that startup is successful.
+    # Cloud Workstations will let us SSH in as soon as the Dockerfile setup has completed, even if
+    # our setup script (and by extension, `install.sh`) has not yet finished running. Luckily, the
+    # setup script is configured to leave a `.sentry.workstation.remote` dotfile in the `~/`
+    # directory for the workstation when this script has run, so we just need to poll until this
+    # occurs (or otherwise timeout).
+    ssh_started_at = datetime.now().timestamp()
+    result = _sync_gcloud_workstation_cmd(
+        [
+            "workstations",
+            "ssh",
+            name,
+            "--command",
+            "[ -f /home/user/.sentry.workstation.remote ] && echo 'Y' || echo 'N'",
+        ],
+        scope=conf,
+        timeout=15,
+        # `ssh` args to silence terminal output.
+        passthrough=[
+            "-o",
+            "UserKnownHostsFile=/dev/null",
+            "-o",
+            "LogLevel ERROR",
+        ],
+        silence_stderr=True,
+    )
+    if result.returncode != 0:
+        raise WorkstationsException(f"Unable to establish SSH connection with workstation {name}")
+
+    output_lines = result.stdout.strip().split("\n")
+    last_line = output_lines[-1] if output_lines else ""
+    if last_line != "Y" and last_line != "N":
+        raise WorkstationsException(
+            f"Could not confirm that workstation {name} started up successfully"
+        )
+    if last_line == "N":
+        _notify(f"Workstation {name} finishing installation, this may take up to 15 minutes...")
+        time_to_sleep = POLLING_INTERVAL - (datetime.now().timestamp() - ssh_started_at)
+        if time_to_sleep > 0:
+            time.sleep(time_to_sleep)
+            return _connect(name, conf, started_at)
+
+    # Everything has started up successfully! Create a detached process to manage the TCP tunnel.
+    localhost_port = _get_open_port()
+    subprocess.Popen(
+        [
+            GCLOUD,
```

Comment:
nit: Would prefer "gcloud" over GCLOUD since it is more clear what this is doing

Reply:
Changed it to `GCLOUD_BIN`.